### DISCUSSION TITLE
如何在开启 OpenClash 后仍然允许路由器本机查询本地 DNS

### DISCUSSION THREAD

**Discussion by Journalist-HK** [vernesong/OpenClash#3400] [Q&A]
我使用 Fake IP 模式，一旦开启 OpenClash，路由器本机就无法使用 nslookup 等命令查询 UDP DNS 服务器。我尝试修改过这个页面上的各种设置，都没有效果。是否需要手动编辑防火墙规则？应该如何写？

![image](https://github.com/vernesong/OpenClash/assets/50702502/15a8d1f2-98a3-4672-b36e-38316529b7f9)

**Comment by hapood** [comment]
我也遇到了这个问题，开启openclash后ping 114.114.114.114也是不通的，但是ping 114.114.115.115可以通。
虽然ping 114.114.115.115可以通，但是nslookup baidu.com 114.114.115.115依旧返回超时。
感觉像是openclash硬编码的限制。

**Comment by Rangtian** [comment]
Dashboard 里的 DNS 查询不行么？

![image](https://github.com/user-attachments/assets/d44a262b-bd2e-41ec-92f2-534e5dc27bd3)
